vortex-array/src/array/implementation.rs

```diff
@@ -72,6 +75,19 @@ impl<A: ArrayImpl + 'static> Array for A {
         ArrayImpl::_vtable(self)
     }
 
+    fn find_kernel(&self, compute_fn: &dyn ComputeFn) -> Option<KernelRef> {
+        let any = compute_fn.as_any();
+
+        // Check each of the known compute functions.
+
+        if any.is::<Filter>() {
+            return <Self as ArrayComputeImpl>::FILTER;
```

Comment:
should this fallback as well? The default might not be populated

Reply:
What do you mean fall back? To what?

Reply:
That happens on line 88

Reply:
but this returns if the constant is none

Reply:
Oh gotcha, yep good catch